src/js/node/assert/AssertionError.hx

```diff
@@ -24,7 +24,9 @@ package js.node.assert;
 typedef AssertionErrorOptions = {
 	@:optional var actual:Dynamic;
 	@:optional var expected:Dynamic;
+	#if (haxe_ver < 4)
```

Comment:
I guess this is the best you can do here for now without https://github.com/HaxeFoundation/haxe/issues/7722.

Reply:
sigh... well we can have a fancy abstract or something, i'll look into this a bit later today